packages/front-end/pages/datasources/[did].tsx

```diff
@@ -58,6 +58,13 @@ const DataSourcePage: FC = () => {
         method: "PUT",
         body: JSON.stringify(updates),
       });
+      apiCall<{ id: string }>("/experiments/import", {
+        method: "POST",
+        body: JSON.stringify({
+          datasource: dataSource.id,
+          force: true,
+        }),
+      });
```

Comment:
This `updateDataSourceSettings` function is used as the onSave handler for a bunch of different things, most of which do not affect the import query.  For some users, the import query is really expensive to run, so we want to avoid running it unnecessarily.

The only place that really needs to refresh is after saving `ExperimentAssignmentQueries`

Reply:
added a quick comparison of queries before calling `/experiments/import`